terraform/environments/tst/main.tf

```diff
@@ -30,5 +30,5 @@ module "deploy" {
   source          = "../../modules/configs/deploy-all"
   aws_account_id  = data.aws_ssm_parameter.aws_account_id.value
   environment     = local.environment
-  webcms_root_url = "https://webdev-cms.crowncommercial.gov.uk/" # Pre-prod
+  webcms_root_url = "https://webuat-cms.crowncommercial.gov.uk/" # Pre-prod
```

Comment:
Why is this changed to UAT? As per the bug raised(https://crowncommercialservice.atlassian.net/browse/SFAT-376) we don't want it to be UAT.